PropertyChanged.Fody/WarningChecker.cs

```diff
@@ -67,7 +67,7 @@ public void EmitWarning(string message)
 
     public void EmitConditionalWarning(ICustomAttributeProvider member, string message)
     {
-        if (SuppressWarnings)
+        if (SuppressWarnings || SuppressOnPropertyNameChangedWarning)
```

Comment:
That doesn't look like the right place to check for the new flag. I'd check it at the call sites instead, as this is a generic method to emit all kinds of warnings.

Reply:
You're right. This was a rushed change. Fixed now.